### PR TITLE
Ers 274 test update

### PR DESCRIPTION
**Pull request by robertkrantz**

### CODE REVIEW

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

File: unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/activity/ActivityJmsIT.java [hunk 1]
```diff
@@ -81,48 +86,104 @@ public void createFishingReport() throws Exception {
...
+		Thread.sleep(10000);
+
+		List<FishingActivityReportDTO> fishingReportsForTripId = getFishingReportsForTripId(newTripId);
+		assertEquals(8, fishingReportsForTripId.size()); // We only expect 8 reports since 4 out of 12 are sub activities to a fishing operation
+
+		for (FishingActivityReportDTO fishingActivityReportDTO : fishingReportsForTripId) {
+			List<FluxReportIdentifierDTO> uniqueIdList = fishingActivityReportDTO.getUniqueFAReportId();
+			FluxReportIdentifierDTO fluxReportIdentifierDTO = uniqueIdList.get(0);
+			String fluxReportId = fluxReportIdentifierDTO.getFluxReportId();
+			assertTrue(generateReportIdList.contains(fluxReportId));
```
Comment (psjobergprivat) [inline unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/activity/ActivityJmsIT.java:128]: Maybe do some "stickprov" on some documents, like 145kg COD catch and similar

File: unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/activity/ActivityJmsIT.java [hunk 2]
```diff
@@ -108,10 +108,17 @@ public void createFishingReport() throws Exception {
 
 		messageHelper.sendMessage("UVMSActivityEvent", reportMessageRequest);
 
+		List<FishingActivityReportDTO> fishingReportsForTripId = getFishingReportsForTripId(newTripId);
+
+		int numberOfFishingActivityReports = fishingReportsForTripId.size();
+
 		// Wait for reports to be persisted
-		Thread.sleep(10000);
+		while (numberOfFishingActivityReports == 0) {
```
Comment (psjobergprivat) [inline unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/activity/ActivityJmsIT.java:116]: So much better! Together with test timeout it is definitely more efficient and robust.

**Review by psjobergprivat — APPROVED** [review]

**Comment by robertkrantz** [comment]
test me

**Review by RobinMilding — APPROVED** [review]

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Review by RobinMilding — APPROVED** [review]

**Pull request merged**